packages/core/data-transfer/types/utils.d.ts

```diff
@@ -86,7 +90,7 @@ export type IProviderTransferResults = {};
 export type ISourceProviderTransferResults = {};
 export type IDestinationProviderTransferResults = {};
 
-export type { KnexTransaction };
+export type KnexTransaction = Knex.Transaction;
```

Comment:
Can we just remove this and use Knex.Transaction everywhere? Is there a case where we might need to extend it or something?